src/graphql/client.js

```diff
@@ -0,0 +1,17 @@
+import { ApolloClient, InMemoryCache } from "@apollo/client"
+import { relayStylePagination } from "@apollo/client/utilities"
+
+const client = new ApolloClient({
+  uri: "https://swapi-graphql.netlify.app/.netlify/functions/index",
+  cache: new InMemoryCache({
+    typePolicies: {
+      Query: {
+        fields: {
+          allPeople: relayStylePagination(),
```

Comment:
I like you found the `relayStylePagination` utility, but we need to get comfortable managing the cache ourselves, so I'm gonna ask that you do so here

Reply:
Since we changed the fetch policy to `network-only`, I'll just leave this code here for now, so the pagination works. Until we come back later to manage the cache manually

Reply:
You shouldn't be doing this at all